app/components/Poll/Poll.css

```diff
@@ -1,185 +1,151 @@
-.pollGraph {
-  animation: graph 1.2s cubic-bezier(41%, 80%, 40%, 94%);
-  background-color: var(--lego-red-color);
-  padding-left: 8px;
-  border-radius: 0 2px 2px 0;
-  font-style: italic;
-  font-weight: 300;
-  color: var(--color-white);
-  height: 30px;
+.contentWrapper {
+  overflow-y: hidden;
+  width: 100%;
+  transition: 0.5s height;
 }
 
-.fullGraph {
-  background-color: #e7e7e7;
-  width: 100%;
+.voteOptionsWrapper {
   display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  height: fit-content;
 }
 
-html[data-theme='dark'] .fullGraph {
+.voteButton,
+.voteButton + .voteButton {
   color: var(--color-white);
-  background-color: var(--color-mono-gray-5);
-}
-
-html[data-theme='dark'] .pollGraph {
-  color: var(--color-black);
-}
-
-.pollGraph span {
-  vertical-align: middle;
-}
-
-@keyframes graph {
-  from {
-    width: 1px;
-  }
-
-  to {
-    width: 100%;
-  }
-}
-
-.pollHeader {
-  border-radius: 8px;
-  margin-bottom: 20px;
-  margin-left: 20px;
-  font-size: 16px;
-  color: var(--lego-font-color);
+  background: var(--lego-red);
+  font-weight: 500;
+  width: 90%;
+  height: 2rem;
+  padding: 5px;
+  margin: 2px;
 }
 
-.voteButton {
-  background: var(--lego-red-color);
-  color: var(--lego-color-gray-light);
-  border: 1px solid var(--border-gray);
+.voteOptions {
   width: 100%;
-  margin: 0 !important;
-  font-size: 15px;
-  max-width: 400px;
+  padding-top: 35px;
 }
 
-.voteButton:hover {
-  opacity: 0.8;
-}
-
-html[data-theme='dark'] .voteButton {
-  color: var(--color-dark-gray-3);
+.bottomInfoWrapper {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
 }
 
-.moreOptionsLink {
-  justify-content: space-between;
+.totalVotesInfo {
+  display: flex;
 }
 
-.arrow {
-  margin-top: 9px;
-  cursor: pointer;
-
-  &:hover {
-    transform: scale(1.5);
-    color: var(--color-red-3);
-    transition: transform 0.2s;
-  }
+.resultsHiddenInfo {
+  font-style: italic;
 }
 
-.blurContainer {
-  display: none;
-  position: absolute;
-  justify-content: center;
+.pollTable {
   width: 100%;
-  height: 100%;
+  font-size: 14px;
 }
 
-.blurOverlay {
-  position: absolute;
-  z-index: 2;
-  color: var(--color-black);
-  margin-top: 25px;
+.pollTable tr {
+  width: 100%;
 }
 
-.optionWrapper:hover .blurContainer {
-  display: flex;
+.pollTable td {
+  border: 0;
+  padding: 5px;
 }
 
-.optionWrapper:hover .blurEffect {
-  filter: blur(3px);
-  pointer-events: none;
+.pollTable .textColumn {
+  text-align: right;
+  line-height: 16px;
+  word-wrap: break-word;
+  width: fit-content;
+  max-width: 200px;
+  padding-right: 13px;
+  border-right: 3px solid #c5c5c5;
 }
 
-.blurArrow {
-  margin-top: 40px;
+.pollTable .graphColumn {
+  width: auto;
+  min-width: 150px;
+  padding-left: 13px;
+  padding-right: 15px;
 }
 
-.alignItems {
+.fullGraph {
   display: flex;
-  justify-content: center;
+  background-color: var(--color-mono-gray-5);
+  color: #000;
+  word-wrap: break-word;
+  width: 100%;
```

Comment:
In general, please group your CSS better: everything regarding display/position/size etc. at the top, and things related to the content (colors, fonts, shadows) at the bottom. The position (not just the CSS `position` property) of an element is often the most tricky part to get right and usually has the most "impact", and should therefore come first. Makes it easier to read and develop. You usually group well, but it's nice to be consistent.

Reply:
thanks for pointing out, it's always nice to read files with consistency ^:heart: